### PR TITLE
services/horizon: Horizon 2.0.0 CHANGELOG

### PR DESCRIPTION
**Pull request by bartekn**

### CODE REVIEW

File: services/horizon/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,23 @@ file. This project adheres to [Semantic Versioning](http://semver.org/).
...
 
+## v2.0.0
+
+### Before you upgrade
+
+Please read the [Captive Core](https://github.com/stellar/go/blob/master/services/horizon/internal/docs/captive_core.md) doc which contains new requirements and migration guide.
+
+### Captive Stellar-Core
+
+Introducing the release candidate with Captive Stellar-Core mode enabled by default. Captive mode relaxes Horizon's operational requirements. It allows running Horizon without a fully fledged Core instance and, most importantly, without a Core database. More information about this new mode can be found in [Captive Core](https://github.com/stellar/go/blob/master/services/horizon/internal/docs/captive_core.md) doc.
```
Comment (ire-and-curses) [inline services/horizon/CHANGELOG.md:16]: Remove reference to release candidate
We can update the release notes after the fact to add a link to the blog post as well when it's live

File: services/horizon/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,23 @@ file. This project adheres to [Semantic Versioning](http://semver.org/).
...
+
+If you run into issues please check [Known Issues](https://github.com/stellar/go/blob/master/services/horizon/internal/docs/captive_core.md#known-issues) or [report an issue](https://github.com/stellar/go/issues/new/choose). Please ask questions in [Keybase](https://keybase.io/team/stellar.public) or [Stack Exchange](https://stellar.stackexchange.com/).
+
+### Breaking changes
```
Comment (ire-and-curses) [inline services/horizon/CHANGELOG.md:20]: Under breaking changes I think it's worth calling out explicitly that the database set up has changed, your old stellar-core config won't work, and emphasise again that **everyone** needs to read the migration guide.

**Review by ire-and-curses — APPROVED** [review]
Some small changes suggested

**Comment by bartekn** [comment]
@ire-and-curses do you want to check the changes: https://github.com/stellar/go/pull/3429/commits/37132af0c6197061981aa2ef552aba95bd8b00c1?

**Comment by bartekn** [comment]
@ire-and-curses merged, should be easy to update if you have further comments.

**Comment by ire-and-curses** [comment]
Looks great! :tada:

**Pull request merged**